homeassistant/components/mullvad/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""The Mullvad VPN integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+from mullvad_api import MullvadAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["binary_sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Mullvad VPN integration."""
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: dict):
+    """Set up Mullvad VPN integration."""
+
+    async def async_get_mullvad_api_data():
+        with async_timeout.timeout(10):
+            api = await hass.async_add_executor_job(MullvadAPI)
+            return api.data
+
+    hass.data[DOMAIN] = update_coordinator.DataUpdateCoordinator(
+        hass,
+        logging.getLogger(__name__),
+        name=DOMAIN,
+        update_method=async_get_mullvad_api_data,
+        update_interval=timedelta(minutes=1),
+    )
+    await hass.data[DOMAIN].async_refresh()
+
+    hass.config_entries.async_update_entry(entry, data={**entry.data})
```

Comment:
This line doesn't make sense, there is no change of the entry data up until this point so why update it?

Reply:
This is basically left in because there used to be many other sensors that would need to be updated, but i was told i was only allowed to have a single sensor for a new integration. 🤷‍♂️
After the merge, i hope to submit a new pull request with the additional sensors.

Reply:
So please remove it if it doesn't do anything

Reply:
Removed.